app/components/layers/useCapitalProjectsLayer.client.tsx

```diff
@@ -47,6 +60,11 @@ export function useCapitalProjectsLayer() {
     autoHighlight: true,
     highlightColor: [129, 230, 217, 218],
     pickable: true,
+    getFilterCategory: (f: Feature<Geometry, CapitalProjectProperties>) => [
```

Comment:
We don't want to pollute the value here to include anything not in the tile itself if we can help it. With the rest of the suggestings here, I was able to set this back to:
```
getFilterCategory: (f: Feature<Geometry, CapitalProjectProperties>) =>
      f.properties.managingAgency,
```